dbt/adapters/impala/connections.py

```diff
@@ -265,10 +285,25 @@ def add_query(
             configuration = {"paramstyle": "format"}
             cursor.execute(sql, bindings, configuration)
 
+            elapsed_time = time.time() - pre
+
+            query_status = str(self.get_response(cursor))
```

Comment:
This will always be status OK. If so we can remove this.

Reply:
Keeping it as reference, I guess we will need to figure out how to get execution status anyways.